pcsx2/gui/FrameForGS.cpp

```diff
@@ -183,13 +185,19 @@ void GSPanel::DoResize()
 	else if( arr > 1 )
 		viewport.y = (int)( (double)viewport.y/arr + 0.5);
 
+	if (g_Conf->GSWindow.ScaleToImageSize && image_width > 0 && image_width < client.GetWidth() &&
+		image_height > 0 && image_height < client.GetHeight() && g_Conf->GSWindow.AspectRatio != AspectRatio_Stretch) {
+			viewport.x = image_width;
+			viewport.y = (int)((double)image_width / targetAr);
```

Comment:
So, I don't get how this is scaled to image size. Wouldn't scaled to image size mean
`viewport.y = image_height;`?